server.js

```diff
@@ -5,14 +5,19 @@ const cors = require("cors");
 const app = express();
 const { initDb } = require('./db');
 const utils = require('./utils');
+const admin = require("firebase-admin");
+const serviceAccount = require('./serviceAccountKey.json');
+
 // Middleware
 app.use(express.json()); // read documentation on what this does
 app.use(cors());
 
-(async () => { 
+(async () => {   
   await initDb();
   await utils.loadRolesIntoMemory();
-
+  admin.initializeApp({
+        credential: admin.credential.cert(serviceAccount),
```

Comment:
let's add loading credentials for Firebase from AWS Parameter Store, too (depending on whether is ENVIRONMENT envvar is set to 'local')